dimension_c: 20
research_altitude: 1
takeoff_altitude: 1
edge_tolerance: 0.1
research_enabled: true
aggressive_enabled: true
preventive_enabled: true
simulation: false
max_altitude: 2
min_step: 0.3
max_step: 2
has_reach_destination_threshold: 0.5
interaction_altitude: 0.1
